WooCommerce/Order.cs

```diff
@@ -212,6 +212,12 @@ public class Order
         /// </summary>
         [DataMember(EmitDefaultValue = false)]
         public Customer customer { get; set; }
+
+        /// <summary>
+        /// Dictionary of order metadata/custom fields. Undocumented.

```

Comment:
Would you please mention in the comment that the order metadata/custom fields when only return when filter[meta] set to true? Thanks.